### ISSUE TITLE
Rexegg (source website) does not want people to copy this content

### ISSUE THREAD

**Issue by jensenr30** [palucki/regexcheatsheet#2]
The author wants to limit the spread of this information. Therefore, we should compile our *own* list of useful regular expressions and release it under the GPLv2 License!

Of course, the first step would be to do a search to see if such a project already exists under a good Free license like the GPL...

EDIT: here is the indication (from rexegg.com) that the author wants to restrict the flow of this information:
![image](https://user-images.githubusercontent.com/3083835/57410678-4eaa4b00-71a0-11e9-9449-e834865a7e35.png)

**Comment by palucki** [comment]
Yes, it looks like we do not receive any support from the rexegg :(
Such a research will be great.
